packagingbuild/centos8/Dockerfile

```diff
@@ -0,0 +1,101 @@
+FROM centos:8
+
+# Download tools
+RUN yum -y install \
+    ca-certificates \
+    curl \
+    wget
+
+RUN yum -y install \git \
+    mercurial \
+    openssh \
+    subversion
+
+    
+RUN yum install -y yum-utils \
+    && dnf config-manager --enable PowerTools \
+    && yum install -y epel-release \
+    && yum -y install \
+    ImageMagick \
+    ImageMagick-devel \
+    libyaml-devel \
+    && dnf config-manager --disable PowerTools \
+    && yum remove -y epel-release yum-utils
+    
+
+# Build tools
+RUN yum -y install \
+    autoconf \
+    automake \
+    bzip2 \
+    bzip2-devel \
+    file \
+    gcc \
+    gcc-c++ \
+    glib2-devel \
+    glibc-devel \libcurl-devel \
+    libevent-devel \
+    libffi-devel \
+    libjpeg-devel \
+    libtool \
+    libwebp-devel \
+    libxml2-devel \
+    libxslt-devel \make \
+    mysql-devel \
+    ncurses-devel \
+    openssl-devel \
+    patch \
+    postgresql-devel \
+    readline-devel \
+    sqlite-devel \
+    xz \
+    xz-devel \
+    zlib-devel
+
+# St2 package build debs
+RUN yum -y install \
+    openldap-devel
+
+# Enable remote pubkey access
+RUN mkdir /root/.ssh && chmod 700 /root/.ssh && \
+    echo "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCdCmmPjsOBWRXc+PKdgDRrsciNjp25zTacyz8Gdkln2ma046brOYXAphhp/85DKgHtANBBt3cl4+HnpDbmAfyq2qZT7hWzAbMxtq0Sj+yyFyUdreXoe4gEKyxpV6o8p/R/XzEcawvqX/vFc5EIFmvTdamxZs9DQmOE5AZMzUB18Kerkrb0/arUcZ8iMi9Ng9a18avow+7oUFZ6Oub7ISz/dkIRojaKO/2paJZ4p+v7ZLn7Hq8TUeBkgAlx872oh8J8linhIq17zK6x4MGL8qiurp2hnfe0cuCxwcsYGy+4DfK51+E2vks6FprCIfF5hIdz26euPn67/YpM0F0b5nXF busybee@drone" >> /root/.ssh/authorized_keys
+
+# Create busybee credentials and make busybee pkey available for root
+COPY busybee*  /root/.ssh/
```

Comment:
Looks like the dir is missing this `busybee` SSH private key that needs to copied to the image.

See https://github.com/StackStorm/st2packaging-dockerfiles/pull/69/files for example.

Reply:
The key is used to SSH between the containers for example when one container trigers the build inside another container, transfer artifacts, etc.

Reply:
Gotcha. Resolved